components/private/ThumbTrack/src/MDCThumbView.m

```diff
@@ -75,18 +73,6 @@ - (void)layoutSubviews {
       [UIBezierPath bezierPathWithRoundedRect:self.bounds cornerRadius:_cornerRadius].CGPath;
 }
 
-- (BOOL)pointInside:(CGPoint)point withEvent:(__unused UIEvent *)event {
```

Comment:
Removing this seems unrelated to the PR's description. 
This logic ensures that the thumbnail is accessible. Please elaborate why we are removing it.

Reply:
MDCThumbTrack is the superview of MDCThumbView and handles all the touch events and has it's hit area expanded. 

Having this subview have an expanded hit area that isn't handling the events makes the exclusive logic fail. Without this change setting exclusiveTouch on MDCSlider will make the material slider not respond when other controls are active, but won't stop other controls from responding when the MDCSlider is active. You can see this if you try the "MDCSlider and UISlider Compared" example.

Reply:
Dialing in @randallli's feedback here with a question: with this logic removed, is the tap target for the thumbtrack still at least 44pts along both the horizontal and vertical axis?

Reply:
Yes, MDCThumbTrack still expands the tap and drag targets to 48px in MDCThumbTrack. https://github.com/material-components/material-components-ios/blob/develop/components/private/ThumbTrack/src/MDCThumbTrack.m#L34

Reply:
No, the code in MDCThumbTrack.m that I linked to is nearly identical to the code I'm deleting in MDCThumbView.m. I was meaning to point out that the logic isn't being lost.

The code in MDCThumbTrack.m actually expands the hit area.

Reply:
Got it! Thank you for clarifying.

Reply:
I can't find a difference between including this code and removing it.  I'd prefer to make the smallest change necessary to achieve the Issue's goal to minimize any unintended side effects.

Can you please provide steps to reproduce the problematic scenario you're describing? Here's what I attempted:

1. Configured the first Slider in SliderCollectionViewController to `exclusiveTouch = YES`
1. Opened the main Demo of MDCSlider (SliderCollectionViewController)
1. Used multi-point touch to tap and touch+drag the first and second Sliders. Alternated between having the `exclusiveTouch` slider receive the first and second touches.

**Expected result**
Removing the code in MDCThumbView would cause a difference in behavior.

**Actual result**
I observed no difference in behavior. Here is what I found in my testing:
* If multi-point single tap was used, the Slider with the first touch responded.
* If multi-point touch-and-drag was used, the Slider with `exclusiveTouch = NO` responded.

Testing of this PR with and without the changes to MDCThumbView

|Without Changes to MDCThumbView|With Changes to MDCThumbView|
|---|---|
|![slider-thumbview-nochange](https://user-images.githubusercontent.com/1753199/46869603-5c5e4480-cdfa-11e8-9851-201eb7c6b8ed.gif)|![slider-thumbview-change](https://user-images.githubusercontent.com/1753199/46869624-6bdd8d80-cdfa-11e8-9441-55a4db986e81.gif)|